src/benchmark_register.cc

```diff
@@ -159,6 +159,7 @@ bool BenchmarkFamilies::FindBenchmarks(
         instance.use_manual_time = family->use_manual_time_;
         instance.complexity = family->complexity_;
         instance.complexity_lambda = family->complexity_lambda_;
+        instance.statistics = &family->statistics_;
```

Comment:
But then looking at https://github.com/google/benchmark/blob/d70417994a3c845c49c4443e92b26a52b320a759/src/benchmark.cc#L619-L633
(which is seemingly the only consumer of `FindBenchmarksInternal()` -> `BenchmarkFamilies::FindBenchmarks()` that we are talking about here) i think the current answer should be **no**.

Unless of course the consumer calls `ClearRegisteredBenchmarks()` in the middle of benchmarking, but then that probably won't the only thing that will break. I'm not sure there are any current locks that could prevent that.